timemanagercontrol.py

```diff
@@ -463,6 +507,7 @@ def updateTimePositionFromSliderPct(self, pct):
         except:
             # extents are not set
             realEpochTime = 0
+            return # as we set the timeframe start to 1970-0-0 below :-(
 
```

Comment:
ok, I get your point, we can just return after the except (no setting a variable that we're not using and no unhappy smileys xD )